core/src/banking_stage.rs

```diff
@@ -1737,12 +1737,11 @@ impl BankingStage {
             return None;
         }
 
-        let tx = SanitizedTransaction::try_create(
-            deserialized_packet.versioned_transaction().clone(),
+        let tx = SanitizedTransaction::try_new(
```

Comment:
tiny nit: replacing `try_create` with `try_new` misled me for a sec. Their comments below make their function clear, but the naming is not as clear.

Reply:
I wasn't able to come up with a good name for this yet.. I have some future refactorings planned that should clean this up later

Reply:
could just omit it in favor of the `TryFrom` impl.  I'm not sure it adds value on its own